qiime/plugin/testing.py

```diff
@@ -84,3 +90,25 @@ def assertSemanticTypeRegisteredToFormat(self, semantic_type, exp_format):
             obs_format, exp_format,
             "Expected semantic type %r to be registered to format %r, not %r."
             % (semantic_type, exp_format, obs_format))
+
+    def transform_format(self, source_format, target, filenames):
+        # Guard any non-QIIME2 Format sources from being tested
+        self.assertTrue(issubclass(source_format, FormatBase))
+
+        transformer = self.get_transformer(source_format, target)
+
+        for filename in filenames:
```

Comment:
This existing loop is applicable for `DirectoryFormat`, but needs to be updated to accommodate `FileFormat`